Doc/library/string.rst

```diff
@@ -469,7 +469,10 @@ of a number respectively. It can be one of the following:
 |         | this option is not supported.                            |
 +---------+----------------------------------------------------------+
 
-For a locale aware separator, use the ``'n'`` presentation type instead.
+For a locale-aware separator, use the ``'n'`` presentation type instead.
+Note that the locale setting for numeric values must first be set using
+the :mod:`locale` module, for example,
+``locale.setlocale(locale.LC_NUMERIC, 'en_US')``.
```

Comment:
Why refer to this here? At most it just belongs in the `n` section.

Reply:
When I was troubleshooting why this wasn't working for me, I searched this page in the docs for "locale aware" and "locale-aware" &ndash; this was the only hit, so I put this here as well to make sure that all spots a person may learn about this in the docs include all the information to use it directly. Most people won't read the entire page in the docs before using something.